scripts/archive/berkeley_migration.sh

```diff
@@ -0,0 +1,675 @@
+    echo ""
+    echo "Notes:"
+    echo "  1. After run migrated data will be filled with migrated blocks till last block in source db"
+    echo "  2. Two logs files will be generated for archiving/debugging operations"
+    echo "  3. Replayer checkpoints (migration-replayer-XXX.json) will be generated"
+    echo ""
+    exit 0
+}
+
+function final(){
+    if [[ ${#} == 0 ]]; then
+        final_help;
+    fi
+
+	local __batch_size=10
+	local __mainnet_archive_uri=''
+	local __migrated_archive_uri='' 
+    local __genesis_ledger=''
+    local __replayer_checkpoint=''
+	local __blocks_bucket=''
+	local __network='' 
+    local __checkpoint_interval=1000
+    local __fork_state_hash=''
+	
+    while [ ${#} -gt 0 ]; do
+        error_message="Error: a value is needed for '$1'";
+        case $1 in
+			-h | --help ) 
+				final_help;
+  			;;
+            -g | --genesis-ledger )
+                __genesis_ledger=${2:?$error_message}
+				shift 2;
+            ;;
+            -f | --fork-state-hash )
+                __fork_state_hash=${2:?$error_message}
+				shift 2;
+            ;;
+            -r | --replayer-checkpoint )
+                __replayer_checkpoint=${2:?$error_message}
+				shift 2;
+            ;;
+            -i | --checkpoint-interval )
+                __checkpoint_interval=${2:?$error_message}
+				shift 2;
+            ;;
+            -s | --source-db )
+                __mainnet_archive_uri=${2:?$error_message}
+                shift 2;
+            ;;
+            -t | --target-db )
+                __migrated_archive_uri=${2:?$error_message}
+                shift 2;
+            ;;
+			-b | --blocks-bucket )
+                __blocks_bucket=${2:?$error_message}
+                shift 2;
+            ;;
+			-bs | --blocks-batch-size )
+                __batch_size=${2:?$error_message}
+                shift 2;
+            ;;
+			-n | --network )
+                __network=${2:?$error_message}
+                shift 2;
+            ;;
+            * )
+                echo -e "${RED} !! Unknown option: $1${CLEAR}\n";
+				echo "";
+				initial_help;
+                exit 0;
+            ;;
+        esac
+    done
+    
+    if [ -z $__mainnet_archive_uri ]; then
+		echo ""
+        echo "Source db not defined"
+		exit 1
+	fi
+
+    if [ -z $__migrated_archive_uri ]; then
+		echo ""
+        echo "'-t|--target-db' argument is not provided. Please provide valid connection string"
+        echo "to existing and initially migrated database. If you don't have such schema please refer to help in 'initial' subcommand" 
+		exit 1
+	fi
+
```

Comment:
The replayer checkpoint is required, right?

```suggestion
    if [ -z $__replayer_checkpoint ]; then
		echo ""
        echo "Replayer checkpoint not defined. Please provide path to latest replayer checkpoint file"
		echo "which should be generated on previous incremental run or inital"
		exit 1
	fi
```